src/Post_Meta_Command.php

```diff
@@ -97,18 +97,86 @@ protected function get_metadata( $object_id, $meta_key = '', $single = false ) {
 	 *
 	 * @param int    $object_id  ID of the object metadata is for
 	 * @param string $meta_key   Metadata key
-	 * @param mixed $meta_value  Optional. Metadata value. Must be serializable
-	 *                           if non-scalar. If specified, only delete
-	 *                           metadata entries with this value. Otherwise,
-	 *                           delete all entries with the specified meta_key.
-	 *                           Pass `null, `false`, or an empty string to skip
-	 *                           this check. For backward compatibility, it is
-	 *                           not possible to pass an empty string to delete
-	 *                           those entries with an empty string for a value.
+	 * @param mixed  $meta_value  Optional. Metadata value. Must be serializable
+	 *                            if non-scalar. If specified, only delete
+	 *                            metadata entries with this value. Otherwise,
+	 *                            delete all entries with the specified meta_key.
+	 *                            Pass `null, `false`, or an empty string to skip
+	 *                            this check. For backward compatibility, it is
+	 *                            not possible to pass an empty string to delete
+	 *                            those entries with an empty string for a value.
 	 *
 	 * @return bool True on successful delete, false on failure.
 	 */
 	protected function delete_metadata( $object_id, $meta_key, $meta_value = '' ) {
 		return delete_post_meta( $object_id, $meta_key, $meta_value );
 	}
+
+	/**
+	 * Cleans up duplicate post meta values on a post.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <id>
+	 * : ID of the post to clean.
+	 *
+	 * <key>
+	 * : Meta key to clean up.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     wp post meta clean-duplicates 1234 enclosure
```

Comment:
Can we add a comment above this example explaining what it does?

Reply:
Done.